Objects/interface/EventBase.h

```diff
@@ -71,6 +72,12 @@ namespace panda {
     //! Trigger object collection with additional features.
     HLTObjectStore triggerObjects = HLTObjectStore("triggerObjects");
 
+    //! Repeatable random number generator
+    RRNG* rng{0};
+
+    //! Initialize the random number generator
+    void initRNG(int N = 50) { delete rng; rng = new RRNG(0, N); }
```

Comment:
One big reason I would want default initialization (of size 0) is to prevent the interface of the `RRNG` from leaking into `EventBase`.